tests/test_Model.py

```diff
@@ -1050,3 +1050,16 @@ def test_model_solves_multi_trip_instance():
     assert_({2} in visit_sets)
     assert_({3, 4} in visit_sets)
     assert_({5, 6} in visit_sets)
+
+
+def test_instance_with_multi_trip_and_release_times(mtvrptw_release_times):
+    """
+    TODO
+    """
+    m = Model.from_data(mtvrptw_release_times)
+    res = m.solve(stop=MaxIterations(50))
+    print(res)
+
+    # OPT is 1068.7 - see table EC.9, p. ec25 of the online supplement
+    # (here: https://doi.org/10.1287/trsc.2022.1161).
+    assert False
```

Comment:
This triggers an assert (cost after move != cost before + delta). I'll see if I can fix that tomorrow.

Reply:
I have distilled a test case. The problem's in `TripRelocate`, which makes sense.